src/Components/Blueprints/ImportBlueprintModal.test.tsx

```diff
@@ -24,106 +24,122 @@ vi.mock('@unleash/proxy-client-react', () => ({
 }));
 
 const BLUEPRINT_JSON = `{
-    "customizations": {
-      "files": [
-      ],
-      "kernel": {
-      },
-      "openscap": {
-      },
-      "packages": [
-        "aide",
-        "sudo",
-        "audit",
-        "rsyslog",
-        "firewalld",
-        "nftables",
-        "libselinux"
-      ],
-      "services": {
-        "enabled": [
-          "crond",
-          "firewalld",
-          "systemd-journald",
-          "rsyslog",
-          "auditd"
-        ]
-      },
-      "subscription": {
-      }
+  "customizations": {
+    "packages": [],
+    "subscription": {
+      "activation-key": "",
+      "base-url": "",
+      "insights": false,
+      "organization": 0,
+      "server-url": ""
+    }
+  },
+  "description": "Lorem ipsum dolor 2211 sit amet, consectetur adipiscing elit. Pellentesque malesuada ultricies diam ac eleifend. Proin ipsum ante, consequat vel justo vel, tristique vestibulum lorem. Vestibulum sit amet pulvinar orci. Vivamus vel ipsum.",
+  "distribution": "rhel-8",
+  "metadata": {
+    "exported_at": "2024-07-29 17:26:51.666952376 +0000 UTC",
+    "parent_id": "b3385e6d-ecc4-485c-b33c-f65131c46f52"
```

Comment:
I just changed this valid blueprint to add the metadata and test it. We also should not have image_requests and id there. They would not matter, however, this is a test for success with a correct blueprint.